solutions/secure/catalogValidationValues.json.template

```diff
@@ -0,0 +1,7 @@
+{
+  "ibmcloud_api_key": $VALIDATION_APIKEY,
+  "region": "us-south",
+  "tags": $TAGS,
+  "instance_name": $PREFIX,
+  "resource_group_name": $PREFIX
```

Comment:
what about the other required variables?
`kms_key_crn` and `existing_kms_instance_guid`